pki/certs.go

```diff
@@ -101,23 +101,29 @@ type CertificatePair struct {
 	Cert    *x509.Certificate
 }
 
-// IssueTestCertificate uses the monitor's certIssuer and certIssuerKey to generate
-// a precertificate and a matching final leaf-certificate that can be submitted
-// to a log. The certificate's subject common name will be a random subdomain
-// based on the certificate serial under the provided baseDomain (or
-// defaultTestCertDomain domain if baseDomain is empty).
+// IssueTestCertificate uses the monitor's certIssuer and certIssuerKey to
+// generate a precertificate and a matching final leaf-certificate that
+// can be submitted to a log. The certificate's subject common name will
+// be a random subdomain based on the certificate serial under the
+// provided baseDomain (or defaultTestCertDomain domain if baseDomain is
+// empty).
 //
-// If windowStart is nil the certificate NotBefore will be set to the current
-// time based on the provided clock. If windowStart is not nil then the
-// certificate NotBefore will be set to the windowStart plus one day.
-
-// If windowEnd is nil the certificate NotAfter will be set to 90 days after the
-// current time based on the provided clock. If windowEnd is not nil then the
-// certificate NotAfter will be set to the windowEnd minus one day.
+// If windowStart is nil the certificate NotBefore will be set to the
+// current time based on the provided clock.
+//
+// If windowEnd is nil the certificate NotAfter will be set to 90 days
+// after the current time based on the provided clock.
+//
+// If windowStart and windowEnd are not nil then issue a 90 day
+// certificate that falls in the window.
 //
-// This function creates certificates that will be submitted to public logs and
-// so while they are not issued by a trusted root  we try to avoid cablint
-// errors to avoid requiring log monitors special-case our submissions.
+// with a NotAfter of `windowEnd - 1 hour` and a NotBefore
+// 90 days earlier.
```

Comment:
Looks like some weird formatting ended up here? I think this is supposed to be all one sentence.

Reply:
Thanks! That was old cruft from when I was using far to many words in the comment. Fixed!